src/components/NoteItem.tsx

```diff
@@ -9,24 +9,24 @@ interface Props {
 const NoteItem: React.FC<Props> = ({ note, handleOpen }) => {
   return (
     <div key={note.id} className="note-item">
-      <div className="note-item-info">
-        <h3>{note.title}</h3>
-        <p>
+      <div className="note-item-info"         onClick={() => {
+          handleOpen();}}>
+        <h3 className='note-item-title'>{note.title}</h3>
+        {/* <p> */}
```

Comment:
It's better to define `note-item-content` here imo
```suggestion
        <div className='note-item-content'>
```